AndroidSDKTests/src/test/java/com/leanplum/LeanplumTest.java

```diff
@@ -359,6 +359,37 @@ public void onResponse(boolean success) {
     assertTrue(Request.userId().equals(userId));
   }
 
+  @Test
+  public void testStartWithVariantDebugInfo() {
```

Comment:
for unit tests, it's easier to understand/maintain the test if the test name is a complete sentence:

```
public void shouldStartAndReturnVariantDebugInfo() {
```

then have comments in the Given/When/Then style (https://www.agilealliance.org/glossary/gwt):

```
// Given the variant debug info flag is enabled.

// When we call start.

// Then the request contains includeVariantDebugInfo parameter.
```